kirin/piv/model_maker.py

```diff
@@ -197,6 +197,25 @@ def _check_stop_time_consistency(previous_rt_stop_time_dep, current_rt_stop_time
         raise InvalidArguments("invalid feed: stop_point's({}) time is not consistent".format(uic8))
 
 
+def _get_message(arret):
+    arrival_stop = get_value(arret, "arrivee", nullable=True)
+    departure_stop = get_value(arret, "depart", nullable=True)
+    motif = None
+    if departure_stop:
+        motif = get_value(departure_stop, "motifModification", nullable=True)
+    if not motif and arrival_stop:
+        motif = get_value(arrival_stop, "motifModification", nullable=True)
+    if not motif and departure_stop:
+        event = get_value(departure_stop, "evenement", nullable=True)
+        if event:
+            motif = get_value(event, "texte", nullable=True)
```

Comment:
minor: Like you did for other nullable fields, you can use get:
```suggestion
        event = departure_stop.get("evenement", {}).get("texte", None)
```

Reply:
done in 666e77bb3b7382474597d6a1f577eab3a2ee0eae